pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1450,31 +1450,31 @@ cdef class _Timedelta(timedelta):
         --------
         >>> td = pd.Timedelta('1 days 2 min 3 us 42 ns')
```

Comment:
I've noticed that `pd.Timedelta(1, unit='T')` doesn't warn - presumably it should?

Reply:
I think you could raise a warning in `parse_timedelta_unit`, and remove `'t'` (and the other deprecated ones) from `timedelta_abbrevs`?

Reply:
Yes, you are right, I checked `pd.Timedelta('1 days 2 T 3 U')` - no warning. I think we should get the warning.

Reply:
thank you, I'll do that.